### PR TITLE
Fix wording about character picker in help

### PR DESCRIPTION
**Pull request by cpeel**
Remove outdated paragraph in the proofreading interface help that discusses the old character selection UI and try to clarify the new character picker interface.

### CODE REVIEW

File: faq/prooffacehelp.php [hunk 1]
```diff
@@ -670,18 +670,12 @@ function echo_row( $name, $tooltip, $button_image_base, $accelerator )
 
 <p>
 As an alternate option, the Proofreading interfaces have a tiled character
-picker section. Hovering over a character will show an enlarged view of it
-in the box on the left and a tooltip text will appear describing the
+picker section. Selecting one of the menus from the top row will show an
+array of characters. Hovering over a character will show an enlarged view
+of it in the box on the left and a tooltip text will appear describing the
 character. Clicking on a character will insert it into the text.
 </p>
```
Comment (srjfoo) [inline faq/prooffacehelp.php]: Good changes. Should we also mention that hovering over the menu tiles will show a description of the content of each menu?

Comment (wf49670) [reply faq/prooffacehelp.php]: Yes, good changes. But I agree that mentioning the tooltip for the tiles would be useful.

Comment (70ray) [reply faq/prooffacehelp.php]: We could change the tooltips for the letter menu tiles to say something like `A variants` and so on.

Comment (srjfoo) [reply faq/prooffacehelp.php]: Yes, also a good idea.

Comment (cpeel) [reply faq/prooffacehelp.php]: Added tooltip mentions and added revised tooltips.

Comment (70ray) [reply faq/prooffacehelp.php]: There is a typo. hoving for hovering.

Comment (cpeel) [reply faq/prooffacehelp.php]: Good catch - thanks!

**Pull request merged**